### ISSUE TITLE
an example (CPS application )of configuring the MAC address forwarding database

### ISSUE THREAD

**Issue by jimmymagemtek** [open-switch/opx-cps#104]
Hi 
The web claims that there is an example (no matter it is C or Python):

MAC address forwarding database (FDB)
The OPX software provides a CPS API/data model for configuring and managing the MAC forwarding database using the CPS API. See CPS application templates for an example of how to access and configure the MAC address forwarding database.
(https://github.com/open-switch/opx-docs/wiki/Layer-2)
But I found no such example but generic template by following its link.

In other words, I need an example to know how to access the existing objects (like forwarding tables, ACL tables) provided by CPS framework.

Please advise. Thanks.

Jimmy

**Comment by jeff-yin** [comment]
There should now be Python-based examples here:
https://github.com/open-switch/opx-tools/blob/master/bin/opx-config-mac
And here:
https://github.com/open-switch/opx-tools/blob/master/bin/opx-show-mac

I believe this should resolve the question, but we still need to update the Layer-2 wiki page.
